src/app/manage/manage-users/manage-users.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="d-flex bd-highlight">
+
+<div class="p-2 w-50 bd-highlight">
+    <div *ngFor="let user of userList">
+        <div class="d-flex flex-column justify-content-center">
+            <div [ngStyle]="user.isDeleted === false ? styleActive : styleDeactive" class="text-primary font-weight-bold">
```

Comment:
use shorthand in ternary.

You should never do something === true or something === false. use shrthand